src/Diagnostics.RuntimeHost/Services/ResourceService/StampService.cs

```diff
@@ -114,7 +114,7 @@ protected virtual string GetTenantIdQuery(string stamp, string startTimeStr, str
             return
                 $@"{tableName}
                 | where TIMESTAMP >= datetime({startTimeStr}) and TIMESTAMP <= datetime({endTimeStr})
-                | where PublicHost =~ ""{stamp}.cloudapp.net"" or PublicHost matches regex ""{stamp}([a-z{{1}}]).cloudapp.net""
+                | where PublicHost =~ ""{stamp}.cloudapp.net"" or PublicHost matches regex ""{stamp}([a-z{{1}}]).cloudapp.net"" or PublicHost matches regex ""{stamp}-([a-z0-9\\.]+).cloudapp.azure.com""
```

Comment:
nitpik: Can you create a const string in this method that just labels what this regex is doing. Something like `const string vmssRegexPublicHost` and if you can also do that for the megastamp regex too just so its readable.

Reply:
Added the strings